core-bundle/src/Resources/contao/dca/tl_files.php

```diff
@@ -252,7 +252,8 @@
 					'title'           => 'maxlength="255"',
 					'alt'             => 'maxlength="255"',
 					'link'            => array('attributes'=>'maxlength="255"', 'dcaPicker'=>true),
-					'caption'         => array('type'=>'textarea')
+					'caption'         => array('type'=>'textarea'),
+					'license'         => array('attributes'=>'maxlength="255"', 'dcaPicker'=>true)
```

Comment:
Why did you add the picker here?

Reply:
https://github.com/contao/contao/pull/3076#issuecomment-858550272